tests/dummy/config/environment.js

```diff
@@ -3,19 +3,15 @@
 module.exports = function (environment) {
   let ENV = {
     modulePrefix: 'dummy',
-    podModulePrefix: 'dummy/pods',
     environment,
     rootURL: '/',
-    locationType: 'auto',
+    locationType: 'history',
     EmberENV: {
       FEATURES: {
         // Here you can enable experimental features on an ember canary build
-        // e.g. 'with-controller': true
-      },
-      EXTEND_PROTOTYPES: {
-        // Prevent Ember Data from overriding Date.parse.
-        Date: false,
+        // e.g. EMBER_NATIVE_DECORATOR_SUPPORT: true
       },
+      EXTEND_PROTOTYPES: false,
```

Comment:
When I reintroduce the documentations, I will be assuming that we will use native JavaScript methods from now on.